Assets/Scripts/Models/Character/Character.cs

```diff
@@ -577,4 +582,18 @@ private void LoadStats()
 
         Debug.ULogChannel("Character", "Initialized " + stats.Count + " Stats.");
     }
+
+    /// <summary>
+    /// Use the stats of the character to determine various traits.
+    /// </summary>
+    private void UseStats()
+    {
+        // Base character speed on their dexterity.
+        speed = Convert.ToSingle(stats["Dexterity"].Value) / 2;
+
```

Comment:
I had some thoughts and thought it would be cool if QuillCorp was the one who asked us to go on our "whatever" expedition or something. And if we made them enough money, met some sort of mandate (I'm thinking Ceaser III type thing), or some other trigger: we could send someone to get trained at their HQ for either something like a stat increase, or maybe "Gun Trained" which boosts accuracy. I agree this is for something way down the road, but i wanted to start implementing parts of it in to the 2.0 milestone.